File: Iniciante/URI 1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const pi = 3.14159;

var raio = parseFloat(lines.shift());
var volume = (4.0 / 3) * pi * Math.pow(raio, 3.0);

console.log('VOLUME = ' + volume.toFixed(3));

File: Iniciante/URI 1011.hs
import Text.Printf

main :: IO ()
main = do
    raio' <- getLine
    let raio = read raio' :: Double
    let volume = (4.0 / 3) * 3.14159 * (raio ** 3.0)
    putStrLn $ printf "VOLUME = %.3f" volume